src/types.ts

```diff
@@ -125,24 +125,23 @@ export const isObject = (o: {}): boolean =>
 /**
  * Return an object filtering out keys that point to undefined values.
```

Comment:
I'd add a note that this method does NOT filter out undefined values inside arrays (only inside objects or objects nested inside arrays)